scipy/spatial/_spherical_voronoi.py

```diff
@@ -250,32 +185,15 @@ def _calc_vertices_regions(self):
         Tyler Reddy, Ross Hemsley and Nikolai Nowaczyk
         """
 
-        # perform 3D Delaunay triangulation on data set
-        # (here ConvexHull can also be used, and is faster)
+        # get Convex Hull
         self._tri = scipy.spatial.ConvexHull(self.points)
 
-        # add the center to each of the simplices in tri to get the same
-        # tetrahedrons we'd have gotten from Delaunay tetrahedralization
-        # tetrahedrons will have shape: (2N-4, 4, 3)
-        tetrahedrons = self._tri.points[self._tri.simplices]
-        tetrahedrons = np.insert(
-            tetrahedrons,
-            3,
-            np.array([self.center]),
-            axis=1
-        )
-
-        # produce circumcenters of tetrahedrons from 3D Delaunay
-        # circumcenters will have shape: (2N-4, 3)
-        circumcenters = calc_circumcenters(tetrahedrons)
+        # triangles will have shape: (2N-4, 3, 3)
+        triangles = self._tri.points[self._tri.simplices]
 
-        # project tetrahedron circumcenters to the surface of the sphere
-        # self.vertices will have shape: (2N-4, 3)
-        self.vertices = project_to_sphere(
-            circumcenters,
-            self.center,
-            self.radius
-        )
+        # get circumcenters of Convex Hull triangles from facet equations
+        # circumcenters will have shape: (2N-4, 3)
+        self.vertices = self.radius * self._tri.equations[:, :-1] + self.center
```

Comment:
@rtshort This is where something tricky / smart is happening, right? So, we're avoiding a bunch of linear algebra stuff now.

Reply:
This is it.

The Qhull FAQ states:
> If the points lie on the unit sphere, the facet normals are the Voronoi vertices [via S. Fortune].

I couldn't find a proof for this anywhere, so I derived it myself.  Here is a proof from an old paper of mine:

![theorem](https://user-images.githubusercontent.com/5646170/63236874-86984400-c20d-11e9-8636-567a10a06e30.png)

Reply:
continuing...

The `equations` attribute contains a hyperplane equation for each facet, in the format [normal, offset].  We only want the normal, which we get using `self._tri.equations[:, :-1]`.

The question remains: which of the two unit normals should we use?  The correct circumcentre is the one which is projected outwards from the facet when viewed from the interior of the convex hull.  After I coded this up, I noticed that Qhull always orients the hyperplane normal outwards.

To finish up, we just need an appropriate scaling and translation.